src/Compiler/SyntaxTree/SyntaxTreeOps.fs

```diff
@@ -1070,3 +1070,18 @@ let (|Get_OrSet_Ident|_|) (ident: Ident) =
     if ident.idText.StartsWithOrdinal("get_") then ValueSome()
     elif ident.idText.StartsWithOrdinal("set_") then ValueSome()
     else ValueNone
+
+let addEmptyMatchClause (mBar1: range) (mBar2: range) (clauses: SynMatchClause list) =
+    let rec addOrPat (pat: SynPat) =
+        match pat with
+        | SynPat.As(lhsPat, rhsPat, range) ->
+            SynPat.As(addOrPat lhsPat, rhsPat, range)
+        | _ ->
+            let mPat1 = mBar1.EndRange
+            let pat1 = SynPat.Wild(mPat1)
+            SynPat.Or(pat1, pat, unionRanges mPat1 pat.Range, { BarRange = mBar2 })
```

Comment:
For my education, were there any other options rather than Or synpat here?

AFAIU the idea here is to fill in the "phantom" match clause with something, so just wondering if this could be something else.

Reply:
My initial idea was to just put an "empty" clause in there, but then I've realised that making Or pattern would be more inline with other cases in the language, like this:
```fsharp
match () with
| _
| _ -> ()
```

Reply:
Alright, yeah why not. Thanks for the explanation!